src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -366,26 +366,70 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
-function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
+function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedElement) {
     $(".tooltip").hide();
-    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    var $clickedElement = $(clickedElement);
+    var panelBody = $clickedElement.parent().find('div[class^="panel-body"]');
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
-    $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+    
+    // If the content is already loaded, toggle the chevron and exit.
+    if ($clickedElement.hasClass("loaded")) {
+        $clickedElement.siblings(".collapse").clearQueue().collapse("toggle");
+        toggleChevron(clickedElement);
+        
+        return;
+    }
+    
+    $clickedElement.find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+    
     panelBody.load(url, function( response, status, xhr ) {
-        if (status == "success") {
-            panelBody.removeClass('hidden');
+        if (status !== "success") {
+            panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
+        } else {
             updateBadgeForPendingComments(panelBody.children(":first").text());
             panelBody.children(":first").remove();
             registerResponseCommentsEvent();
             registerCheckboxEventForVisibilityOptions();
             enableHoverToDisplayEditOptions();
-      } else {
-          panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
-          panelBody.removeClass('hidden');
-      }
-      $(sender).find('div[class^="placeholder-img-loading"]').html("");
+        }
+        
+        //clearQueue to clear the animation queue to prevent animation build up
```

Comment:
This comment is not really clear as it is hard to even see where clearQueue is called. Place it right above the line with clearQueue?
The loaded class should only be added in the success branch of the ajax request, as reloading should be allowed if there is an error. Ideally, the error message should have a retry button to allow the user to load again, but that is for another issue.